### PR TITLE
Fix bug: Incorrect error msg for returncheckin

### PR DESCRIPTION
**Pull request by alyssa-savier**

### CODE REVIEW

**Review by NicolasCwy — CHANGES_REQUESTED** [review]
Hi could you remove the PDFs that you have committed, as well as check if your tests are passing?

**Comment by alyssa-savier** [comment]
> Hi could you remove the PDFs that you have committed, as well as check if your tests are passing?

Removed, and they are passing! just checkout test case not passing and i think jeremy fixed it!

**Comment by alyssa-savier** [comment]
oops accidentally closed the PR

File: docs/UserGuide.md [hunk 1]
```diff
@@ -217,6 +217,15 @@ Example:
...
   list.
 
+<div markdown="block" class="alert alert-info">
+**:information_source: Notes:**<br>
+
+* Guests with the same names and emails as other guests that are already checked in are allowed.
+
+* Guests with either the same passport numbers or same room numbers as guests that are already checked in are not allowed.
+
+* Guests with the same passport number as a past guest that is written in the archived are not allowed. If the guest is a returning guest, you should use returncheckin instead.
```
Comment (rgbpokka) [inline docs/UserGuide.md:227]: typo in the word archive

**Review by rgbpokka — CHANGES_REQUESTED** [review]
Other than minor typo, LGTM

**Review by NicolasCwy — CHANGES_REQUESTED** [review]
image

There's still a few PDFs that need to be removed!

File: docs/UserGuide.md (hunk 1 above)
Comment (alyssa-savier) [reply docs/UserGuide.md]: Fixed!

**Comment by alyssa-savier** [comment]
> image
> 
> There's still a few PDFs that need to be removed!

Removed!

**Review by NicolasCwy — APPROVED** [review]
LGTM!

**Pull request merged**